BridgeCareApp/BridgeCare/DataAccessLayer/SimulationAnalysis.cs

```diff
@@ -0,0 +1,69 @@
+﻿using BridgeCare.ApplicationLog;
+using BridgeCare.Interfaces;
+using BridgeCare.Models;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.DataAccessLayer
+{
+    public class SimulationAnalysis : ISimulationAnalysis
+    {
+        public SimulationAnalysis()
+        {
+        }
+
+        public SimulationAnalysisModel GetSimulationAnalyis(int simulationId, BridgeCareContext db)
+        {
+            try
+            {
+                var simulations = db.SIMULATIONS
+                    .Where(b => b.SIMULATIONID == simulationId)
+                    .Select(b => new SimulationAnalysisModel()
+                    {
+                        simulationId = b.SIMULATIONID,
+                        committed_start = b.COMMITTED_START,
+                        committed_period = b.COMMITTED_PERIOD,
+                        analysis = b.ANALYSIS,
+                        budget_constraint = b.BUDGET_CONSTRAINT,
+                        benefit_limit = b.BENEFIT_LIMIT,
+                        comments = b.COMMENTS,
+                        jurisdiction = b.JURISDICTION,
+                        benefit_variable = b.BENEFIT_VARIABLE
```

Comment:
not on UI as of this change but there is a pending request to add it.

Reply:
ok.